WordPress/src/main/java/org/wordpress/android/ui/stats/refresh/utils/DateUtils.kt

```diff
@@ -49,8 +49,17 @@ class DateUtils @Inject constructor(
         )
     }
 
-    fun formatDateRange(from: Long, to: Long): String =
-            DateUtils.formatDateRange(contextProvider.getContext(), from, to, DateUtils.FORMAT_ABBREV_MONTH)
+    fun formatDateRange(from: Long, to: Long, timezone: String): String {
+        val formatter = java.util.Formatter(StringBuilder(50), localeManagerWrapper.getLocale())
```

Comment:
Np: We can import `java.util.Formatter` and remove full qualifier `java.util.`.

Reply:
Fixed in https://github.com/wordpress-mobile/WordPress-Android/pull/13655/commits/cb4722acfdade60db6aeb70df4cc735dc110c850